source/basic/error-try-catch/src/error-cause/index.js

```diff
@@ -0,0 +1,27 @@
+// 数字の文字列を二つ受け取り、合計を返す関数
+function sumNumStrings(a, b) {
+    try {
+        const aNumber = safeParseInt(a);
+        const bNumber = safeParseInt(b);
+        return aNumber + bNumber;
+    } catch (e) {
+        throw new Error("Failed to sum a and b", { cause: e });
+    }
+}
+
+// 数値の文字列を受け取り数値を返す関数
+// 'text' など数値にはならない文字列を渡された場合は例外を投げられる
+function safeParseInt(numStr) {
+    const num = parseInt(numStr, 10);
```

Comment:
```suggestion
    const num = Number.parseInt(numStr, 10);
```

Number.parseInt に統一してるので、 `Number.parseInt` へ変更
https://jsprimer.net/basic/implicit-coercion/#string-to-number